### PR TITLE
Fix package dependency on ENV variable.

### PR DESCRIPTION
**Pull request by rheinardkorf**

### CODE REVIEW

File: src/audit/tide/tide.go [hunk 1]
```diff
@@ -10,18 +10,17 @@ import (
 	"fmt"
 	"strings"
 	"reflect"
-	"github.com/xwp/go-tide/src/env"
```
Comment (valendesigns) [inline src/audit/tide/tide.go:13]: Only removing this one line seems off. Wouldn't we just import the new repo, or each package in the repo as needed?
```
https://github.com/wptide/pkg
```
OR
```
https://github.com/wptide/pkg/env
https://github.com/wptide/pkg/log
```

File: .env.dist [hunk 2]
```diff
@@ -5,6 +5,9 @@ TIDE_API_PROTOCOL=http
 TIDE_API_KEY=<YOUR_API_KEY>
 TIDE_API_SECRET=<YOUR_API_SECRET>
 TIDE_API_VERSION=v1
+# User in Tide API to be credited for project.
+# Omit to user authenticated client user.
+TIDE_API_CREDITED_USER=<LOGIN_NAME>
```
Comment (valendesigns) [inline .env.dist:10]: So this would be set manually to the `audit_server` user, and if left empty would be what user? I don't see a fallback anymore.

Comment (rheinardkorf) [reply .env.dist:10]: It defaults to the authenticated user if empty. 

```
RequestClient string   `json:"request_client,omitempty"`
```

File: src/audit/tide/tide.go (hunk 1 above)
Comment (rheinardkorf) [reply src/audit/tide/tide.go:13]: Yeah scrap this whole PR.... its fixed in #9 by replacing all these imports completely.

Aside, Just removing that line is correct. We are no longer using `env.GetEnv` its passed as an optional value in the `result` object. If nothing is found, value will be empty (even for fallback) in which case the authenticated user will be used. (which is likely audit_server anyway)

But ignore all this, I'm closing the PR.

Final aside, trust the tests ;)

**Pull request closed without merging**